content/docs/higher-order-components.md

```diff
@@ -279,7 +279,7 @@ This form may seem confusing or unnecessary, but it has a useful property. Singl
 
 ```js
 // Instead of doing this...
-const EnhancedComponent = withRouter(connect(commentSelector)(WrappedComponent))
+const EnhancedComponent = withRouter(connect(commentSelector))(WrappedComponent)
```

Comment:
Previous code was correct. `withRouter` takes a single argument and returns a component: https://v5.reactrouter.com/web/api/withRouter. It doesn't return a function as your code change suggests.